pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -100,6 +100,23 @@ type VSphereConfig struct {
 		VMName string `gcfg:"vm-name"`
 	}
```

Comment:
Balu, during my next task ready new format config file I will remove whatever is not needed.

Changed Datacenter to Datacenters.
Changed Datastore to DefaultDatastore.